src/sentry/eventstore/models.py

```diff
@@ -1,22 +1,30 @@
 from __future__ import absolute_import
 
+import pytz
+
 from datetime import datetime
-from django.utils import timezone
+from dateutil.parser import parse as parse_date
 
 from sentry.models import EventCommon, EventDict
 from sentry.db.models import NodeData
 
+from sentry.snuba.events import Columns
+
 
 def ref_func(x):
     return x.project_id or x.project.id
 
 
 class Event(EventCommon):
-    def __init__(self, project_id, event_id, group_id=None, data=None):
+    def __init__(
+        self, project_id, event_id, group_id=None, message=None, data=None, snuba_data=None
+    ):
         self.project_id = project_id
         self.event_id = event_id
         self.group_id = group_id
+        self.message = message
         self.data = data
+        self._snuba_data = snuba_data or {}
```

Comment:
How long do you expect this to be around ?
Asking because I would like to avoid the concern that the two implementations (this one and the snuba one could diverge)

Reply:
Are you referring to this line or the SnubaEvent class in general? This line will probably stay to hold data fetched from Snuba. SnubaEvent will go away soon, outisde this PR it's only used in a couple of small cases.

Reply:
The snuba event class that has a copy paste of a lot of the logic you added here to the Event class.

Reply:
We'll be able to remove SnubaEvent really soon